content/zh/docs/contribution_guidelines/conduct.md

```diff
@@ -0,0 +1,166 @@
++++ 
+title = "行为规范"
+weight = 480 
++++
+
+<h2 id="about">关于本规范</h2>
+
+<p>
+线上社区会有很多不同背景的人，CUE 贡献者致力于为所有人提供一个友好、安全以及热情的环境，
+不论性别、性取向、是否健全、智力、外貌、身高、种族、国籍、人种、年龄、信仰或其他个人特征。
+</p>
+
+<p>
+行为规范的第一个目的是制定行为的基本标准，这样不同价值观以及不同沟通风格的人们可以有效、高效以及互相尊重地讨论 CUE。
+</p>
+
+<p>
+第二个目的是当社区中发生冲突时提供解决冲突的机制。
+</p>
+
+<p>
+第三个目的是让我们的社区对来自不同背景的人都保持欢迎态度。
+多向性对这个项目至关重要；为了成功，CUE 需要来自不同背景的贡献者和用户。（启发于 <a href="https://blog.golang.org/open-source">Go, Open Source, Community</a>）
+</p>
+
+<p>
+我们相信正向的争论或分歧对于项目以及社区至关重要。但是，不尊重人是绝对不行的。我们重视不同的观点，但是我们重视有礼貌的行为。
+</p>
+
+<h2 id="values">CUE 的价值观</h2>
+
+<p>
+这些是 CUE 社区中应该尊重的价值观。
+</p>
+
+<ul>
+<li>保持友好以及热情
+<li>保持耐心
+    <ul>
+        <li>请谨记人们有不同的交流方式而且也不是所有人都用同一种语言。（话语本身的意思或语气可能会在翻译中丢失）
+    </ul>
+<li>考虑周全
+    <ul>
+    <li>高效的沟通需要多用心，多考虑你的话怎么才能比较好理解。
+    <li>记住有时候不评论是最好的选择。
+    </ul>
+<li>保持尊重
+    <ul>
+    <li>特别是要尊重不同的意见
+    </ul>
+<li>乐于助人
+    <ul>
+    <li>真诚地解释其他人的观点，不要吹毛求疵
+    <li>当确实有不同观点时，要尽量解释原因
+    </ul>
+<li>避免以下行为：
+    <ul>
+    <li>跑题：专注于主题；如果你想要讨论其他事情，请重开新的对话
+    <li>非建设性批评：不要只是谴责当前事态；提供或至少征求改进的建议
+    <li>吐槽(简短、低效、带有恶意的评论)
+    <li>讨论可能冒犯或敏感的问题；这些通常会导致不必要的争执
+    <li>微歧视：口头、行为以及环境的侮辱，传达敌意、贬低或否定的信息对个人或团体进行轻视或侮辱。
+    </ul>
+</ul>
+
+<p>
+人是非常复杂的，肯定会被误解或误解别人。当这些不可避免的冲突发生时，请克制出于自我保护的冲动或推卸责任。
+也请不要挑起无端的冲突，要先假定对方是无意冒犯。即使对方故意挑衅，也不要将争端升级。
+当冲突发生时，<i>所有人</i>都有责任将冲突降级。
+</p>
+
+<h2 id="code">行为规范</h2>
+
+<h3 id="our-pledge">我们的承诺</h3>
+
+<p>
+为了维护一个开放和有好的环境，我们作为贡献者和维护者承诺，项目和社区中的所有人都可以随时提问，
+无论年龄、体型、是否健全、种族、性别、经验、受教育程度、社会经济地位、国籍、外貌、种族、信仰以及性取向。
+</p>
+
+<h3 id="our-standards">我们的标准</h3>
+
+<p>有助于创造积极环境的行为包括：</p>
+
+<ul>
+<li>使用热情和包容的语言</li>
+<li>尊重不同的观点和经历</li>
+<li>优雅地接受建设性的批评</li>
+<li>专注于对社区最有利的事情</li>
+<li>对其他社区成员有同理心</li>
+</ul>
+
+<p>参与者不可接受的行为包括：</p>
+
+<ul> 
+<li>使用包含色情信息的语言或图片，以及令人反感的性暗示</li>
```

Comment:
The [DeepL translation](https://www.deepl.com/en/translator#zh/en/%E4%BD%BF%E7%94%A8%E5%8C%85%E5%90%AB%E8%89%B2%E6%83%85%E4%BF%A1%E6%81%AF%E7%9A%84%E8%AF%AD%E8%A8%80%E6%88%96%E5%9B%BE%E7%89%87%EF%BC%8C%E4%BB%A5%E5%8F%8A%E4%BB%A4%E4%BA%BA%E5%8F%8D%E6%84%9F%E7%9A%84%E6%80%A7%E6%9A%97%E7%A4%BA) doesn't quite capture the original:

> Use of language or images containing pornographic messages and offensive sexual innuendo

The original is quite specific in the second part of the sentence:

> The use of sexualized language or imagery and unwelcome sexual attention or advances

Is there something we can tweak here?